.storybook/stories/DataFiltering/components/DashboardPagination.tsx

```diff
@@ -170,6 +179,13 @@ export const DashboardPagination = ({
 						<Button
 							variant="tertiary"
 							onClick={() => {
+								if (!activeColumns) return;
```

Comment:
This doesn't seem to allow the button to work? This, the Close button and pressing Escape should always just close the Drawer without doing anything and without maintaining any previous state of the inputs for the next time it's opened

Reply:
I probably should have just managed state with `FormData` 😅 

Anyway, I've updated to include the close button, should all be fine now.